src/api/models/question.js

```diff
@@ -212,4 +212,18 @@ export default class QuestionAPI extends APIRepository {
       data
     })
   }
+
+  getLevels(data = {}, cache) {
+    return this.sendRequest({
+      apiMethod: 'get',
+      api: this.api,
+      request: this.APIAdresses.levels,
+      resolveCallback: (response) => {
+        return response.data.data // List of Options Objects [{key,Value}]
```

Comment:
check for option model

Reply:
levels are not compatible with Option Model